CHANGELOG.md

```diff
@@ -5,15 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.5.1] - 2024-04-10
```

Comment:
should this be 1.5.1? I am just asking. We have bumped up the versions whenever there was new pyincore.